packages/components/atoms/f-button/src/components/RouterLinkButton.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+    <router-link
+        :to="to">
```

Comment:
Would this be applied automatically with this? 
https://github.com/justeat/fozzie-components/blob/ebf1eac7099bde14a983807c532adf17bcd1f89f/packages/components/atoms/f-button/src/components/Button.vue#L14

Reply:
Does not seem to work, it breaks the storybook. I assume it is because `to` is not a standard attribute.

Reply:
![Screenshot 2021-06-15 at 16 27 52](https://user-images.githubusercontent.com/26770126/122081151-c5da5080-cdf6-11eb-8f07-5fdc0773d2e0.png)
Hope that helps